### PR TITLE
[GPU] Fix a bug in post-operation optimization

### PR DESCRIPTION
**Pull request by jade-cho**
+ Fix not to change eltwise_round to eltwise_linear for optimization

### CODE REVIEW

File: inference-engine/thirdparty/clDNN/src/graph_optimizer/reorder_inputs.cpp [hunk 1]
```diff
@@ -400,11 +400,32 @@ void insert_reorders_in_dir(program& p, const std::map<program_node*, format::ty
...
+                    }
+                }
+            }
+
+            if (!should_skip_add_reorder) {
+                auto& reorder_node = p.get_or_create(reorder);
+                p.add_intermediate(reorder_node,
+                                *travel_direction_wrapper<dir>::second(node, next),
+                                *travel_direction_wrapper<dir>::first(node, next),
+                                !reorder_pair.second);
```
Comment (lznamens) [inline inference-engine/thirdparty/clDNN/src/graph_optimizer/reorder_inputs.cpp:427]:
```suggestion
                p.add_intermediate(reorder_node,
                                   *travel_direction_wrapper<dir>::second(node, next),
                                   *travel_direction_wrapper<dir>::first(node, next),
                                   !reorder_pair.second);
```

**Comment by lznamens** [comment]
Post-ops optimization fix with eltwises LGTM, thanks for finding it!

**Review by byungilm — APPROVED** [review]

File: inference-engine/thirdparty/clDNN/src/graph_optimizer/reorder_inputs.cpp (hunk 1 above)
Comment (jade-cho) [reply inference-engine/thirdparty/clDNN/src/graph_optimizer/reorder_inputs.cpp]: Applied

**Review by lznamens — APPROVED** [review]
LGTM.

**Review by lznamens — APPROVED** [review]

**Pull request merged**